testing/ef_tests/src/cases/merkle_proof_validity.rs

```diff
@@ -100,7 +100,7 @@ impl<E: EthSpec> LoadCase for KzgInclusionMerkleProofValidity<E> {
                     fork_name
                 )))
             }
-            ForkName::Deneb => {
+            ForkName::Deneb | ForkName::Electra => {
```

Comment:
may be worth adding a TODO(electra) here to update the generic type

Reply:
I actually think this was just a mistake which wasn't picked up since Electra is disabled in the EF tests. [I have added a separate code branch for the Electra block](https://github.com/sigp/lighthouse/pull/5122/commits/ef724dc9e0a63bb574b3a1bf47c78c629b8a973b#diff-e85ac9049ffe91303ecede898312a363d2d80f2484b300d9b5ba8740c6dd0312R109).